src/BannerSDK/Client.php

```diff
@@ -56,26 +54,161 @@ public function __construct($baseUri, $username, $password)
     /**
      * Get all banners.
      *
-     * @return Banner[]|null Banner collection.
+     * @return PaginatedCollection Banner collection.
+     * @throws BannerManagerException
      * @throws GuzzleException
      */
     public function getBanners()
     {
         $response = $this->doRequest('GET', '/api/banners');
 
+        if ($response->getStatusCode() !== 200) {
+            throw new BannerManagerException(
+                $response->getReasonPhrase(),
+                $response->getStatusCode()
+            );
+        }
+
+        $data = json_decode($response->getBody(), true);
+
+        $banners = [];
+
+        foreach ($data as $datum) {
+            $banners[] = new Banner($datum);
+        }
+
+        return new PaginatedCollection($banners, count($banners), 1, 1);
```

Comment:
I guess page and limit should not always be 1 and 1.

Reply:
This is just temporary, currentPage and totPages will be coming from the banner-manager response. For now we do not support pagination so I put current and totalPages to 1